docs/examples/use_cases/webdataset-externalsource.ipynb

```diff
@@ -0,0 +1,576 @@
+{
```

Comment:
<blockquote>stored in abuffer</blockquote><p>stored in a buffer</p><p> </p><p>Maybe intermediate buffer or prefetch buffer?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA/DALI/pull/3153/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>